src/about-menu/open-about-menu.ts

```diff
@@ -0,0 +1,51 @@
+import { BrowserWindow, app, shell } from 'electron';
+import path from 'path';
+
+let aboutWindow: BrowserWindow | null = null;
+
+export function openAboutWindow() {
+	const aboutPath = path.join( __dirname, 'menu', 'about-menu.html' );
+
+	if ( aboutWindow ) {
+		aboutWindow.focus();
+		return;
+	}
+
+	aboutWindow = new BrowserWindow( {
+		width: 284,
+		height: 284,
```

Comment:
[Proposal]: Consider adding width, and height as parameters, to determine window size without the need to altering the component's code. ( To get a better understanding why I propose that I suggest you to [read this](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle) )

Reply:
Hey hey thanks for the suggestion! I tested this approach but it seems to break some types when I try to call the openAboutWindow() and becomes messy quite quickly. I looked how we handle new browser window in other places and it seems that we add the height and width in a way that I initially added them (example:
https://github.com/Automattic/studio/blob/bb8e4c9f669055ee47084c093c5063e00d8aa7a3/src/screenshot-window.ts#L8 ) I think we can keep it as it for now!

I did more the values for the width and height to constants in https://github.com/Automattic/studio/pull/104/commits/77001c083a87d5048d2b5b9b83585c3a1ed745b2